app/javascript/chat/actions/actions.js

```diff
@@ -180,3 +205,21 @@ export function deleteMessage(messageId, successCb, failureCb) {
     .then(successCb)
     .catch(failureCb);
 }
+
+
+export async function getAllMessages(channelId, messageOffset) {
+  const response = await request(`/chat_channels/${channelId}?message_offset=${messageOffset}`, {
+    method: 'GET',
```

Comment:
`'GET'` is the default and `credentials: 'same-origin'` are included automatically when using the `request` utility function. See lines https://github.com/forem/forem/blob/master/app/javascript/utilities/http/request.js#L27 and  https://github.com/forem/forem/blob/master/app/javascript/utilities/http/request.js#L47